src/components/tooltip/tooltip.spec.js

```diff
@@ -1,20 +1,38 @@
-describe('<md-tooltip> directive', function() {
-  var $compile, $rootScope, $material, $timeout, $$mdTooltipRegistry;
+describe('<md-tooltip directive', function() {
+  var $compile, $rootScope, $material, $timeout, $mdPanel, $$mdTooltipRegistry;
   var element;
 
-  beforeEach(module('material.components.tooltip'));
-  beforeEach(module('material.components.button'));
-  beforeEach(inject(function(_$compile_, _$rootScope_, _$material_, _$timeout_, _$$mdTooltipRegistry_){
-    $compile   = _$compile_;
-    $rootScope = _$rootScope_;
-    $material  = _$material_;
-    $timeout   = _$timeout_;
-    $$mdTooltipRegistry = _$$mdTooltipRegistry_;
-  }));
+  // var injectLocals = function($injector) {
+  //   $compile = $injector.get('$compile');
+  //   $rootScope = $injector.get('$rootScope');
+  //   $material = $injector.get('$material');
+  //   $timeout = $injector.get('$timeout');
+  //   $mdPanel = $injector.get('$mdPanel');
+  //   $$mdTooltipRegistry = $injector.get('$$mdTooltipRegistry');
+  // };
+
+  beforeEach(function() {
+    module(
+      'material.components.tooltip',
+      'material.components.button',
+      'material.components.panel'
+    );
+    // inject(injectLocals);
+  });
+
+  beforeEach(inject(function(_$compile_, _$rootScope_, _$material_, _$timeout_,
+      _$mdPanel_, _$$mdTooltipRegistry_){
+        $compile = _$compile_;
+        $rootScope = _$rootScope_;
+        $material = _$material_;
+        $timeout = _$timeout_;
+        $mdPanel = _$mdPanel_;
+        $$mdTooltipRegistry = _$$mdTooltipRegistry_;
```

Comment:
I propose using the `$injector` here

```
$compile = $injector.get('$compile');
```

No need for the ugly underscore prefix / suffix